posthog/src/main/java/com/posthog/PostHog.kt

```diff
@@ -757,12 +757,22 @@ public class PostHog private constructor(
         }
 
         if (config?.sendFeatureFlagEvent == true && shouldSendFeatureFlagEvent) {
-            val props = mutableMapOf<String, Any>()
-            props["\$feature_flag"] = key
-            // value should never be nullabe anyway
-            props["\$feature_flag_response"] = value ?: ""
-
-            capture("\$feature_flag_called", properties = props)
+            remoteConfig?.run {
```

Comment:
I'd use `let` instead
https://kotlinlang.org/docs/scope-functions.html#let
its the right scope for checking nullability